conf/messages

```diff
@@ -2111,4 +2111,19 @@ optout.optOutError.nextUpdates-bullet-1              = go back to your
 optout.optOutError.nextUpdates-link                  = next updates
 optout.optOutError.nextUpdates-bullet-2              = and try again
 optout.optOutError.home-bullet-1                     = go back to the
-optout.optOutError.home-bullet-2                     = homepage
\ No newline at end of file
+optout.optOutError.home-bullet-2                     = homepage
+
+#######################################################################################################################
+## Opt In ##
+#######################################################################################################################
+
+optIn.beforeYouStart.heading                   = Before you start
+optIn.beforeYouStart.desc1                     = Reporting quarterly allows HMRC to give you a more precise forecast of how much tax you owe to help you budget more accurately.
+optIn.beforeYouStart.desc2                     = To report quarterly you will need compatible software. There are both paid and free options for you or your agent to choose from.
+optIn.beforeYouStart.reportQuarterly           = Reporting quarterly
+optIn.beforeYouStart.reportQuarterly.text      = If you choose to opt in:
+optIn.beforeYouStart.reportQuarterly.text.p1   = you will need to submit an update every three months in addition to your tax return.
+optIn.beforeYouStart.reportQuarterly.text.p2   = all your income courses will move to quarterly reporting.
```

Comment:
It looks like there's a small typo of "courses" instead of "sources".

Reply:
I just copied the content from the ticket. I will ask them to update the ticket. Thanks for noticing.